lib/main.js

```diff
@@ -23,95 +112,40 @@ if (flags.get('version')) {
   var diag = require('./diagnostics');
   diag();
 
+} else if (flags.get('devices').length > 0) {
+	
+	var deviceArguments = flags.get('devices');
+	for(var i = 0; i < deviceArguments.length ; i++) {
```

Comment:
also spacing, here:

`for (var i = 0; i < deviceArguments.length; i++)`

(perhaps I should add a travis-ci linter..)